explorer/src/api/index.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  GATEWAYS_API,
+  MIXNODES_API,
+  VALIDATORS_API,
+  BLOCK_API,
+} from './constants';
+
+export class Api {
+  static fetchMixnodes = async () => {
+    try {
+      const res = await fetch(MIXNODES_API);
+      const json = await res.json();
+      return json;
+    } catch (error: any) {
+      console.log('error ', error);
+      throw error.message;
```

Comment:
Rethrow the error and not just the message?

```suggestion
      throw error;
```